packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -1919,7 +1986,7 @@ export function updateRefreshers(
 		crossFieldKeys: change.crossFieldKeys,
 		maxId: maxId as number,
 		revisions,
-		constraintViolationCount,
+		inputConstraintViolationCount,
```

Comment:
needs the output constraint violation count here as well

Reply:
Added